iOS-ReactNative-Build.yaml

```diff
@@ -89,18 +98,19 @@ steps:
   - AC_XCODE_LIST_DIR: "$AC_XCODE_LIST_DIR"
   - AC_XCODE_VERSION: "$AC_XCODE_VERSION"
   outputs:
-  - XCODE_DEVELOPER_DIR_PATH: "XCODE_DEVELOPER_DIR_PATH"
+  - XCODE_DEVELOPER_DIR_PATH: XCODE_DEVELOPER_DIR_PATH
 - componentType: appcircle_cocoapods
-  componentVersion: "1.1.*"
+  componentVersion: 1.1.*
   stepName: Cocoapods Install
   enabled: true
   onError: $exit
   webUrl: https://github.com/appcircleio/appcircle-cocoapods-component
   repoUrl: https://github.com/appcircleio/appcircle-cocoapods-component.git
-  commit: fa877ae
+  commit: 2c80939
   inputs:
   - AC_PROJECT_PATH: "$AC_PROJECT_PATH"
   - AC_REPOSITORY_DIR: "$AC_REPOSITORY_DIR"
+  - AC_COCOAPODS_VERSION: "$AC_COCOAPODS_VERSION"
 - componentType: appcircle_ios_build_version_increment
   componentVersion: 2.0.*
   stepName: Increment Build and Version Number
```

Comment:
@boztopuz can you change increment build version number's commit id:
- 4707781 => a14f5dd